cron/3.queueProcess.nolock.php

```diff
@@ -108,6 +108,11 @@
         $kill['npc'] = isNPC($kill);
         $kill['awox'] = ($kill['npc'] == true) ? false : isAwox($kill);
         $kill['solo'] = ($kill['npc'] == true) ? false : isSolo($kill);
+        
+        $capitalShipTypeIds = array(
+        '23919', '28848', '37604', '23757', '28850', '11567', '34328', '45647', '20185', '42243', '23915', '42242', '671', '20189', '22852', '45649', '3764', '37606', '45645', '37605', '42241', '19724', '19722', '24483', '37607', '28846', '23913', '20187', '19726', '20183', '23773', '19720', '3514', '28844', '28352', '23911', '42132', '42126', '42124', '42125', '42133', '23917', '52907'
+        )
+        $kill['isCapital'] = in_array($kill['shipTypeID'], $capitalShipTypeIds);
```

Comment:
Is `shipTypeID` the correct key here? I believe it is, as other helper functions within this same file access this key on the first item in the `involved` list (i.e. the victim).